frontend/server/aws-helper.test.ts

```diff
@@ -118,6 +118,10 @@ describe('isS3Endpoint', () => {
     expect(isS3Endpoint('s3.cn-north-1.amazonaws.com.cn')).toBe(true);
   });
 
+  it('checks a valid s3 fips GovCloud endpoint', () => {
```

Comment:
+1

Can you add one for private link endpoint or does it have a similar format?

Reply:
Will add one for private-link endpoint, all amazon s3 endpoints follow the same format of having s3 prefix in the string and then amazonaws.com afterwards.